lib/error.js

```diff
@@ -36,10 +36,10 @@ const codes = {
 module.exports = ServiceError;
 
 module.exports.AsServiceError = (err, reporter, code, metadata) => {
-	code = code ? code : codes.InternalServerError;
+	code = (code || code === 0)? code : codes.InternalServerError;
```

Comment:
Please add a space after the closing bracket.

Reply:
Done.